README.md

```diff
@@ -60,7 +60,7 @@ import { h } from 'preact';
 class WithPreact extends withComponent(withPreact()) {
   static get props() {
     return {
-      name: props.string
+      name: props.string // String could be used also to define the prop type
```

Comment:
I'd actually prefer we advocate native usage instead of built-in, and introduce custom usage separately. We can do that in a separate PR.